packages/connect-popup/src/index.tsx

```diff
@@ -79,6 +79,10 @@ const handleMessage = (
                     detail: 'response-event-error',
                     message: data.payload?.error || 'Unknown error',
                 });
+                analytics.report({
+                    type: EventType.ViewChangeError,
+                    payload: { code: data.payload.code },
+                });
```

Comment:
It looks like type-check does not like it, since `data.payload` could be `undefined`. Maybe something like:

```suggestion
                analytics.report({
                    type: EventType.ViewChangeError,
                    payload: { code: data.payload?.code 'Missing code'  },
                });
```

Reply:
thanks, fixed and squashed